packages/types/src/index.ts

```diff
@@ -12,3 +12,4 @@ assertSingletonPackage('@polkadot/types');
 export { default as ContractAbi } from './ContractAbi';
 export * from './codec';
 export * from './index.types';
+export { IExtrinsic } from './types';
```

Comment:
No TypeScript type exports in index.

Reply:
That was @kwingram25 's commit that somehow sneaked in here as the first commit because I somehow didn't branch off master. But I can happily delete it in my branch (might be gone already anyways)